src/modules/profile/profile.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Profile } from "@prisma/client";
+import { Service } from "typedi";
+
+import {
+  CreateProfileInput,
+  UpdateProfileInput,
+} from "../../core/dto/profile.dto.js";
+import { prisma } from "../../prisma/index.js";
+
+@Service()
+export default class ProfileService {
+  async getProfileByUserId(userId: number): Promise<Profile> {
+    const profile = await prisma.profile.findUnique({
+      where: { userId: userId },
```

Comment:
in the `where:` statement, when the variable has the same name as the field, you can eliminate one for repetitiveness. So it would look like `where: { userId }`